Education/CodingStandards.md

```diff
@@ -211,21 +229,21 @@ Please instal both linters:
 
 To activate linters, restart VS code after installation.
 
-## 8. Spell Check
+## 9. Spell Check
 Spelling errors are easy to make, and can lead to hours of debugging.  Please instal the following spell checker extension:
 - Code Spell Checker
 
-## 9. Pull Requests
+## 10. Pull Requests
 Anyone reviewing code in PR's (Pull Requests), should ensure that all standards were abided by before approving the PR.
 
-## 10. Repository Branching
+## 11. Repository Branching
 - Whenever you have a Story, Bug, Task, etc that results in committing code to the repository, you must create a new Branch, and name it after the ticket assigned to you
 - Append onto the branch name with a basic description of the ticket.
 - Example: "Tenant-Build-CPQ-#89_Create-Top-Of-Page-Button"
 
-## 11. Multi Line Code
+## 12. Multi Line Code
 
-### 11.1. Long Strings
+### 12.1. Long Strings
 - Instead of using new line characters  
 
 <span><img src="/Education/media/images/redX.png" width="20" height="20"/></span>
```

Comment:
what is this for?

Reply:
This creates the visual red X when looking at it on the front end